### PR TITLE
Minor bugfix for c2st

### PR DESCRIPTION
**Pull request by michaeldeistler**

### CODE REVIEW

**Comment by psteinb** [comment]
Just by chance, how backwards compatible is this?

**Comment by michaeldeistler** [comment]
Both `axis` and `dim` work for torch.mean. But `dim` is the "usual" way of doing it, [see here](https://pytorch.org/docs/stable/generated/torch.mean.html)

**Review by psteinb — APPROVED** [review]
LGTM ... not that I have much to say. ;-)
